packages/gatsby-cli/gatsby.js

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
```

Comment:
was this supposed to have been added back?

Reply:
yeah, this is the entry, i was trying to just replace the old bin.js with something with the right permissions

Reply:
k, ready to merge and publish again?

Reply:
yup